src/datasets/load.jl

```diff
@@ -47,17 +47,18 @@ const RE_IMAGEFILE = r".*\.(gif|jpe?g|tiff?|png|webp|bmp)$"i
 isimagefile(f) = matches(RE_IMAGEFILE, f)
 const RE_TEXTFILE = r".*\.(txt|csv|json|md|html?|xml|yaml|toml)$"i
 istextfile(f) = matches(RE_TEXTFILE, f)
-
+const RE_TIMESERIESFILE = r".*\.(ts)$"i
```

Comment:
We'll have to figure out how to deal with time series datasets in different formats eventually. This may involve changing the text file detection logic. No need to worry about that now, this is just a note for when that time comes :)